### PR TITLE
Add support for PostgreSQL

### PR DESCRIPTION
**Pull request by malteriechmann**
With this pull request a migration for PostgreSQL ist added. Tested in Flow 3.2.x.

### CODE REVIEW

**Comment by Akii** [comment]
+1

File: Migrations/Postgresql/Version20160512080712.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ */
+class Version20160512080712 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     * @return void
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is autogenerated, please modify it to your needs
```
Comment (afoeder) [inline Migrations/Postgresql/Version20160512080712.php]: remove this line please

File: Migrations/Postgresql/Version20160512080712.php [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        $this->addSql('CREATE TABLE ag_event_domain_model_storedevent (eventid BIGINT NOT NULL, occuredon TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL, event TEXT NOT NULL, PRIMARY KEY(eventid))');
+    }
+
+    /**
+     * @param Schema $schema
+     * @return void
+     */
+    public function down(Schema $schema)
+    {
+        // this down() migration is autogenerated, please modify it to your needs
```
Comment (afoeder) [inline Migrations/Postgresql/Version20160512080712.php]: ditto

**Comment by afoeder** [comment]
👎  see in-line-comments, would you fix that please?

**Comment by malteriechmann** [comment]
Fixed.

**Comment by afoeder** [comment]
Thanks a lot!

**Comment by malteriechmann** [comment]
Well … thank you!

**Pull request merged**